src/api/github/checkRun.test.ts

```diff
@@ -51,12 +50,13 @@ describe('createCheckRun', () => {
 
   beforeEach(() => {
     mocked(Octokit).mockReturnValue(mockClient as unknown as Octokit);
-    mocked(git.listRemotes).mockResolvedValue([
```

Comment:
Can we keep these as-is? I know opinions differ here, but I like to think we gain a bit more integration assurance that way rather than going full unit.

Reply:
sure